.github/workflows/publish.yml

```diff
@@ -0,0 +1,47 @@
+
+name: Publish to PyPI
+
+on:
+  release:
+    types:
+      - published
```

Comment:
@kwilcox this LGTM but if you want to test it in the PR before merging you can change lines 4-7 to:

```yaml
on:
  pull_request:
```

so it runs here. We will need to change it back to releases only before merging.